### PR TITLE
Update dependency dp-mongodb-in-memory

### PR DESCRIPTION
**Pull request by nshumoogum**
### What

Use new version of dp-mongodb-in-memory (1.2.0) - contains breaking changing due to passing context to methods.

Also add .github repo with pull request and issues template

### How to review

Check changes make sense and run `make test`

### Who can review

Anyone

### CODE REVIEW

File: api_feature.go [hunk 1]
```diff
@@ -58,7 +58,7 @@ func (f *APIFeature) RegisterSteps(ctx *godog.ScenarioContext) {
 	ctx.Step(`^I PATCH "([^"]*)"$`, f.IPatch)
 	ctx.Step(`^I DELETE "([^"]*)"`, f.IDelete)
 	ctx.Step(`^the HTTP status code should be "([^"]*)"$`, f.TheHTTPStatusCodeShouldBe)
-	ctx.Step(`^the response header "([^"]*)" should be "([^"]*)"$`, f.TheResponseHeaderShouldBe)
+	ctx.Step(`^the response header "([^"]*)" should be "([^";]*)"$`, f.TheResponseHeaderShouldBe)
```
Comment (rafahop) [inline api_feature.go:61]: Is this change intentional?

Comment (nshumoogum) [reply api_feature.go:61]: Just saw this myself (when looking over), I will remove it

**Review by rafahop — APPROVED** [review]
